app/helpers/language_helper.py

```diff
@@ -30,6 +31,11 @@ def handle_language(metadata: Optional[MetadataProxy] = None) -> None:
         g.allowed_languages = get_allowed_languages(schema_name, launch_language)
         request_language = request.args.get("language_code")
         if request_language and request_language in g.allowed_languages:
+            if cookie_session.get("survey_title"):
+                schema = load_schema_from_metadata(
+                    metadata=metadata, language_code=request_language  # type: ignore
```

Comment:
What is the type ignore for?.

Reply:
I was checking few different solutions here and possibly forgot to remove it after changing the code. Removed now, thanks!